requirements/benchmark.txt

```diff
@@ -0,0 +1,9 @@
+appdirs==1.4.3
+attrs==16.3.0
+click==6.7
+hypothesis==3.7.0
```

Comment:
Given that release tooling will auto-increment things, we should have something else in the housekeeping to also take care of this in benchmark suite.

Shall I open a ticket?